themes/mainroad/assets/css/style.css

```diff
@@ -239,6 +248,11 @@ code {
 	padding: 0 5px;
 }
 
+code:hover {
+	filter: brightness(.85);
+	cursor: pointer;
```

Comment:
Why? It's a bit distracting to hover over a code block in a page and see it change to the hand pointer (for links and buttons), when it is not a button or a link. (It might also give the impression that you can click to copy?)

Also, is dimming the code block necessary? I think it should just be treated as ordinary text.

Reply:
> you can click to copy?

we can do that, actually.

OK, no more experimenting. I'll finalise the pr shortly.